django/thunderstore/repository/tests/test_package_upload.py

```diff
@@ -46,6 +52,87 @@ def test_package_upload(user, manifest_v1_data, community):
     assert version.package.owner == identity
 
 
+@pytest.mark.django_db
+def test_package_upload_missing_privileges(user, manifest_v1_data, community):
+
+    icon_raw = io.BytesIO()
```

Comment:
For future reference, consider creating a pytest fixture out of the icon or other often-used data like this